android/cli/commands/_build.js

```diff
@@ -2675,6 +2676,67 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
 		}
 	}, this);
 
+	tasks.push(done => {
+		// copy js files into assets directory and minify if needed
+		this.logger.info(__('Processing JavaScript files'));
+
+		const inputFiles = [];
+		const copyUnmodified = [];
+		Object.keys(jsFiles).forEach(relPath => {
+			const from = jsFiles[relPath];
+			if (htmlJsFiles[relPath]) {
+				// this js file is referenced from an html file, so don't minify or encrypt
+				copyUnmodified.push(from);
+			} else {
+				inputFiles.push(from);
+			}
+		});
+
+		const task = new ProcessJsTask({
+			inputFiles,
+			incrementalDirectory: path.join(this.buildDir, 'incremental', 'process-js'),
+			logger: this.logger,
+			builder: this,
+			jsFiles: Object.keys(jsFiles).reduce((jsFilesInfo, relPath) => {
+				jsFilesInfo[relPath] = {
+					src: jsFiles[relPath],
+					dest: path.join(this.buildBinAssetsResourcesDir, relPath)
+				};
+				return jsFilesInfo;
+			}, {}),
+			jsBootstrapFiles,
+			sdkCommonFolder: path.join(this.titaniumSdkPath, 'common', 'Resources'),
+			defaultAnalyzeOptions: {
+				minify: this.minifyJS,
+				transpile: this.transpile,
+				sourceMap: this.sourceMaps || this.deployType === 'development',
+				resourcesDir: this.xcodeAppDir,
```

Comment:
I don't think this field will exist on Android...?

Probably should be:
```suggestion
				resourcesDir: this.buildBinAssetsResourcesDir,
```